src/index.js

```diff
@@ -14,13 +14,23 @@ export default (Alpine) => {
             modifiers = modifiers.filter((modifier) => modifier !== 'global')
 
             Mousetrap.bindGlobal(modifiers, ($event) => {
+                const filamentModal = document.querySelector('.fi-modal-open')
```

Comment:
While Livewire-specific code in an Alpine package feels ok, Filament specific code like this feels weird. Can we somehow make this selector generic? Is it something like "does a parent element to this one have a keydown/keyup event listener"? No selector for that but could maybe be done with JS?